extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -269,11 +324,14 @@ export async function collectTargetDatabaseInfo(
 	userName: string,
 	password: string): Promise<TargetDatabaseInfo[]> {
 
-	const connectionProfile = getConnectionProfile(
+	const connectionProfile = getTargetConnectionProfile(
 		targetServer.properties.fullyQualifiedDomainName,
 		targetServer.id,
 		userName,
-		password);
+		password,
+		// when connecting to a target Azure SQL DB, use true/false
```

Comment:
collecting target database table info for SQL DB scenario - verify that true/false is correct for the target

Reply:
cc @siyangMicrosoft

Reply:
true/false is correct